src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_FOR_DAY = 40;
+
+  const MIDDLE_TERM = 3;
+  const MIDDLE_TERM_DISCONT = 20;
+
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCONT = 50;
+
+  if (days >= LONG_TERM) {
+    return RENT_FOR_DAY * days - LONG_TERM_DISCONT;
+  }
+
+  if (days >= MIDDLE_TERM && days < LONG_TERM_DISCONT) {
```

Comment:
First, you are now comparing the number of `days` and the discount (`LONG_TERM_DISCONT`) - and you should have a term (`LONG_TERM`).
Secondly, it is not necessary to check whether `days < LONG_TERM`, because if `days` is greater than or equal to `LONG_TERM` - the function will end its execution on the previous `if`.